go/consensus/tendermint/apps/staking/transactions_test.go

```diff
@@ -90,7 +90,7 @@ func TestReservedAddresses(t *testing.T) {
 	_ = staking.NewReservedAddress(testPK)
 
```

Comment:
Update tests to include checks that return values are correct.

Reply:
Added